packages/office-ui-fabric-react/src/components/Shimmer/ShimmerLine/ShimmerLine.styles.ts

```diff
@@ -33,6 +33,7 @@ export function getStyles(props: IShimmerLineStyleProps): IShimmerLineStyles {
         borderTopStyle: 'solid',
         borderBottomStyle: 'solid',
         borderColor: semanticColors.bodyBackground,
+        borderWidth: 0,
```

Comment:
@Vitalius1 intentional?

Reply:
Yeah. This is to address your comment [here](https://github.com/OfficeDev/office-ui-fabric-react/pull/9922#discussion_r306984503). Was it something else you meant?

Reply:
Ah, I thought this was snapshot output. My bad, I misread.